src/test/java/com/jcabi/github/RtLabelsITCase.java

```diff
@@ -70,9 +69,10 @@ public static void setUp() throws Exception {
         final Github github = new RtGithub(key);
         repos = github.repos();
         repo = repos.create(
-            Json.createObjectBuilder().add(
-                "name", RandomStringUtils.randomAlphanumeric(Tv.TEN)
-            ).add("auto_init", true).build()
+            new Repos.NewRepoSettings(
```

Comment:
@cvrebert maybe add a constructor for this common case of 10 random alphanumeric characters, false and true?